openmc/data/energy_distribution.py

```diff
@@ -777,8 +777,6 @@ def from_endf(cls, file_obj, params):
         efl, efh = params[0:2]
         return cls(efl, efh, tm)
 
-
-
 class DiscretePhoton(EnergyDistribution):
```

Comment:
There should be two lines before a class definition per PEP8
```suggestion

class DiscretePhoton(EnergyDistribution):
```